### DISCUSSION TITLE
The app couldn’t be loaded

### DISCUSSION THREAD

**Discussion by aakashprajapati1** [osiset/laravel-shopify#728] [General]
So issue with ngrok right? I have used a local tunnel as well but also the same issue with the local tunnel. Is there any other way to checking in locally?

**Comment by gnikyt** [comment]
Theres been a lot of people have issues with ngrok, they'd use another method and things would be fine. It may not be in your case, but could be. Without more information I wouldn't know. Make sure your app settings on Shopify are correct.
